src/javascript/views/Login/Index.js

```diff
@@ -49,17 +48,17 @@ class Login extends Component {
                        hintText='email'
                        floatingLabelText='email'
                        defaultValue='john.doe@example.com'
-                       onKeyDown={::this.submit} /><br/>
+                       onKeyDown={this.submit.bind(this)}/><br/>
```

Comment:
binding here creates a new function, it is more efficient to do it in the constructor like:

```javascript
constructor() {
  super();
  this.submit = this.submit.bind(this);
}
```